CorsixTH/Src/th_map.cpp

```diff
@@ -629,7 +637,7 @@ void THMap::setParcelOwner(int iParcelId, int iOwner)
     _updatePurchaseMatrix();
 }
 
-void THMap::_checkDividers(THMapNode* pNode, const THMapNode* pOriginalNode,
+bool THMap::_checkDividers(THMapNode* pNode, const THMapNode* pOriginalNode,
```

Comment:
Commit misses documentation what magic return value means (checkDividers == true, does that mean it's checked, or checked good, or needs fixed, or ... ?